app/components/course-page/setup-step/repository-setup-help.hbs

```diff
@@ -0,0 +1,29 @@
+<div data-test-repository-setup-help class="flex flex-col items-stretch max-w-2xl mt-6 rounded border dark:border-white/10 space-y-2">
+  <div class="border-b border-gray-200 dark:border-white/5 bg-gray-50 dark:bg-gray-700/50 rounded-t pl-3 pr-1 py-1 flex items-center justify-between">
```

Comment:
There's a LOT of custom styling here 🙂 We tend to avoid micro-optimizations like `h-[26px]`, and rely on standard styling like `px-1`, `py-2` etc. We also tend to keep things more consistent across the codebase and only introduce "new" styles of components if absolutely needed. 

I'm also not sure whether this needs to draw that much attention compared to the steps above. 

@andy1li can we just make this a `prose-sm` block like before? My hunch is that we don't need to make this that prominent (we can re-evaluate if we still get a lot of user reports in the future)